packages/table/src/helpers/get-row-appearance.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react'
+import * as tinycolor from 'tinycolor2'
+import {DefaultTheme} from 'styled-components'
+
+export type IntentType = 'success' | 'warning' | 'info' | 'danger' | 'none'
```

Comment:
change to `TableRowIntentType` to avoid confusion
edit: actually use `TableRowIntentType` from `types`